src/tools/fuzzing.h

```diff
@@ -120,7 +120,8 @@ class TranslateToFuzzReader {
     std::cout << "shrink level: " << options.passOptions.shrinkLevel << '\n';
   }
 
-  void build() {
+  void build(bool buildEmitAtomics = true) {
```

Comment:
Looks good!

Only change I'd like is to please replace `build` with `init`, I think that's clearer.

Reply:
(I mean to have `initEmitAtomics`, not the name of the function, which I just noticed is `build`... ;) )